tidb-binlog/pump_client/client.go

```diff
@@ -487,13 +501,12 @@ func (c *PumpsClient) watchStatus(revision int64) {
 		case wresp := <-rch:
 			if wresp.Err() != nil {
 				// meet error, watch from the latest revision.
+				// pump will update the key periodly, it's ok for we to lost some event here
 				log.Warn("[pumps client] watch status meet error", zap.Error(wresp.Err()))
-				rch = c.EtcdRegistry.WatchNode(c.ctx, c.nodePath, revision)
+				rch = c.EtcdRegistry.WatchNode(c.ctx, c.nodePath, 0)
```

Comment:
What does it mean to set revision to 0 here?

Reply:
well, finally it will reach here...
```
// Watch watchs the events of key with prefix.
func (e *Client) Watch(ctx context.Context, prefix string, revision int64) clientv3.WatchChan {
    if revision > 0 {
        return e.client.Watch(ctx, prefix, clientv3.WithPrefix(), clientv3.WithRev(revision))
    }
    return e.client.Watch(ctx, prefix, clientv3.WithPrefix())
}
```
it means will not specify revision to watch, then it will just return the later evens while the Watch reach etcd but not return the evens start at the specified revision.

<img width="822" alt="Screen Shot 2019-08-08 at 11 29 42 AM" src="https://user-images.githubusercontent.com/1681864/62673151-e43bbd80-b9cf-11e9-9f27-7acbf1bd8ed3.png">